README.md

```diff
@@ -2,30 +2,31 @@
 
 [![Build Status](https://travis-ci.org/geerlingguy/ansible-role-certbot.svg?branch=master)](https://travis-ci.org/geerlingguy/ansible-role-certbot)
 
-Installs Certbot (for Let's Encrypt) for RHEL/CentOS or Debian/Ubuntu.
+Installs and configures Certbot (for Let's Encrypt).
 
 ## Requirements
 
-Certbot requires Git to be installed. You can install Git using the `geerlingguy.git` role.
+Certbot requires Git to be installed if one wants to install Certbot from Git repository instead of package management. You can install Git using the `geerlingguy.git` role.
 
 ## Role Variables
 
+    certbot_from_git: no
     certbot_repo: https://github.com/certbot/certbot.git
     certbot_version: master
     certbot_keep_updated: yes
 
-Certbot code repository options. This role clones the agent from the configured repo, then makes the `certbot-auto` script executable.
+Certbot Git repository options. This role clones the agent from the configured repo, then makes the `certbot-auto` script executable if `certbot_from_git` is `yes`. Otherwise it will be installed from distribution's package management.
```

Comment:
Since there are basically two sets of config, and many of these vars are only used in the source install method, I'd rather add a section with a third level heading here and group all the source-specific vars in there.